taxcalc/tests/test_parameters.py

```diff
@@ -330,86 +345,96 @@ def check_ss_earnings_c(ppo, reform, ifactor):
 
 
 def test_create_parameters_from_file(paramsfile):
-    p = Parameters.from_file(paramsfile.name)
-    irates = Parameters._Parameters__rates
-    inf_rates = [irates[2013 + i] for i in range(0, 12)]
-
-    assert_array_equal(p._almdep,
-                       expand_array(np.array([7150, 7250, 7400]), inflate=True,
-                                    inflation_rates=inf_rates, num_years=12))
-    assert_array_equal(p._almsep,
-                       expand_array(np.array([40400, 41050]), inflate=True,
-                                    inflation_rates=inf_rates, num_years=12))
-    assert_array_equal(p._rt5,
-                       expand_array(np.array([0.33]), inflate=False,
-                                    inflation_rates=inf_rates, num_years=12))
-    assert_array_equal(p._rt7,
-                       expand_array(np.array([0.396]), inflate=False,
-                                    inflation_rates=inf_rates, num_years=12))
-
-
-def test_parameters_get_default(paramsfile):
-    paramdata = taxcalc.parameters.default_data()
+    with open(paramsfile.name) as pfile:
+        params = json.load(pfile)
+    ppo = Parameters(parameter_dict=params)
+    irates = Parameters.default_inflation_rates()
+    inf_rates = [irates[ppo.start_year + i] for i in range(0, ppo.num_years)]
+
+    assert_array_equal(ppo._almdep,
+                       expand_array(np.array([7150, 7250, 7400]),
+                                    inflate=True, inflation_rates=inf_rates,
+                                    num_years=ppo.num_years))
+    assert_array_equal(ppo._almsep,
+                       expand_array(np.array([40400, 41050]),
+                                    inflate=True, inflation_rates=inf_rates,
+                                    num_years=ppo.num_years))
+    assert_array_equal(ppo._rt5,
+                       expand_array(np.array([0.33]),
+                                    inflate=False, inflation_rates=inf_rates,
+                                    num_years=ppo.num_years))
+    assert_array_equal(ppo._rt7,
+                       expand_array(np.array([0.396]),
+                                    inflate=False, inflation_rates=inf_rates,
+                                    num_years=ppo.num_years))
+
+
+def test_parameters_get_default():
+    use_classmethod = True
+    if use_classmethod:
+        paramdata = Parameters.default_data()
+    else:  # use legacy global default_data function
+        paramdata = taxcalc.parameters.default_data()
     assert paramdata['_CDCC_ps'] == [15000]
 
 
-def test_update_Parameters_raises_on_no_year(paramsfile):
-    p = Parameters.from_file(paramsfile.name)
-    user_mods = {"_STD_Aged": [[1400, 1200]]}
+def test_reform_with_no_year():
```

Comment:
Do you object to the style of 'test that expects an exception to be raised has "raises" in the name'? I find that it is a helpful idiom to keep it straight in my brain that this is a test wherein, if an exception is *not* raised, the test fails. I have tried to be consistent with such a convention throughout the tests. Now that I've explained the idiom, what are your thoughts on it?